submissions/xT0PERx/OOP exercise/index.js

```diff
@@ -0,0 +1,93 @@
+class Mammals {
+  constructor(props) {
+    this.name = props.name;
+    this.saying = props.saying;
+  }
+  say() {
+    return this.saying;
+  }
+  printInfo() {
+    print(
+      [
+        this.species,
+        this.name,
+        this.gender,
+        this.hands,
+        this.legs,
+        this.say(),
+      ].join('; ')
```

Comment:
Class Mammals doesn't have properties `hands`, `gender` and some other.
Super class shouldn't know anything about child classes internals.
Make sure the props specific to child classes are nadled by relevant child classes.
You may want to benefit from method overloading.